src/chrome/content/rules/rns.online.xml

```diff
@@ -0,0 +1,16 @@
+<ruleset name="rns.online">
+	<target host="rns.online" />
+	<target host="www.rns.online" />
+	<target host="cdn.rns.online" />
+	<target host="m.rns.online" />
+	<test url="http://cdn.rns.online/" />
+	<test url="http://cdn.rns.online/?" />
+	<test url="http://cdn.rns.online/?q" />
+	<test url="http://cdn.rns.online/?s" />
+	<test url="http://cdn.rns.online/#" />
+	<test url="http://cdn.rns.online/#q" />
+	<exclusion pattern="^http://cdn\.rns\.online/?$" />
+	<exclusion pattern="^http://cdn\.rns\.online/?\?" />
+	<exclusion pattern="^http://cdn\.rns\.online/?#" />
```

Comment:
`?` need to be escaped or removed. Exclusions without trailing slash are not allowed.